test/action_controller/json/include_test.rb

```diff
@@ -158,8 +224,14 @@ def test_render_resource_with_deep_include
               ]
             }
           }
+        end
 
-          assert_equal(expected, response)
+        def with_default_includes(include_tree)
+          original = ActiveModel::Serializer.config.default_includes
```

Comment:
You're right. I just had a head scratching moment trying to figure out what is happening here.

It turns out that in normal usage we don't seem to use the default value, but are instead creating a new include tree (see lib/active_model_serializers/adapter/attributes.rb line 6). 

This makes me think that we should either remove the memoized class method (simpler), or ensure that we are using the memoized include tree on instances too (potentially more performant, harder to test as we need to do something like clear the cache on all serializers).

Do you have any preferences? I'm leaning towards removing the memoized version, since it is equivalent to the existing code, but I can implement either version.